sources/packages/backend/apps/db-migrations/src/migrations/1745962287131-AddRelationshipStatusMarriedUnable.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddRelationshipStatusMarriedUnable1745962287131
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Please add the rollback evidence to the PR description.

Reply:
Added.